a10_neutron_lbaas/v1/handler_member.py

```diff
@@ -37,14 +37,15 @@ def _create(self, c, context, member):
                                                c.device_cfg['use_float'])
         server_name = self._meta_name(member, server_ip)
         conn_limit = c.device_cfg['conn-limit']
```

Comment:
Can we have this be "(server|member|)-conn-limit" or similar so we know what "conn-limit" refers to?